### PR TITLE
Update Calling SDK in the quickstart samples to the latest version

### PR DESCRIPTION
**Pull request by petrsvihlik**
## Purpose
Updated to the latest calling SDK to fix a WebRTC standard change affecting Chrome, Edge, and Safari.
https://azure.microsoft.com/en-us/updates/upgrade-to-azure-communication-services-calling-javascript-sdk-110-by-31-july-2021/

## Does this introduce a breaking change?

```
- [ ] Yes
- [x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
- Follow the instructions to run the code in READMEs. (Typically just `npm i` and ` npx webpack-dev-server --entry ./client.js --output bundle.js --debug --devtool inline-source-map`)

## What to Check
- Make sure calls are working.

### CODE REVIEW

**Comment by petrsvihlik** [comment]
@akania @cwhil2492 @alkwa-msft can some of you please review & merge? thanks :)

File: join-calling-to-teams-meeting/package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/communication-calling": "^1.0.1-beta.1",
-    "@azure/communication-common": "^1.0.0"
+    "@azure/communication-calling": "^1.1.0",
```
Comment (cwhil2492) [inline join-calling-to-teams-meeting/package.json:13]: This needs to be latest beta please. Joining teams meeting is a beta feature.

**Review by cwhil2492 — CHANGES_REQUESTED** [review]

File: join-calling-to-teams-meeting/package.json (hunk 1 above)
Comment (petrsvihlik) [reply join-calling-to-teams-meeting/package.json]: good catch! fixed. thanks :)

**Review by cwhil2492 — APPROVED** [review]

**Comment by cwhil2492** [comment]
Hi @petrsvihlik, good to merge?

**Comment by petrsvihlik** [comment]
Go for it.

**Pull request merged**